packages/shared/src/components/sidebar/SquadsList.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { ReactElement } from 'react';
+import { Squad } from '../../graphql/squads';
+import DefaultSquadIcon from '../icons/DefaultSquad';
+import NewSquadIcon from '../icons/NewSquad';
+import { ClickableNavItem } from './ClickableNavItem';
+import { ItemInner, NavItem, SidebarMenuItem } from './common';
+import { Image } from '../image/Image';
+import { cloudinary } from '../../lib/image';
+
+type SquadsListProps = {
+  squads: Squad[];
+  onNavTabClick?: (page: string) => unknown;
+};
+
+const SquadImage = ({ image, name }: Squad) => (
+  <Image
+    title={name}
+    src={image}
+    fallbackSrc={cloudinary.squads.imageFallback}
+    className="object-cover w-5 h-5 rounded-full"
+    loading="lazy"
+  />
+);
+
+export function SquadsList({
+  squads,
+  onNavTabClick,
+}: SquadsListProps): ReactElement {
+  const newSquadMenuItem: SidebarMenuItem = {
+    icon: () => <NewSquadIcon />,
+    title: 'New squad',
+    path: '/new-squad',
+  };
+
+  return (
+    <>
+      {squads.map((squad) => {
+        const { handle, name, permalink, image } = squad;
+        const menuItem: SidebarMenuItem = {
+          icon: () =>
+            image ? <SquadImage {...squad} /> : <DefaultSquadIcon />,
```

Comment:
Image should always be defined. Server side handles the fallback image.
So if user didn't provide an image for the squad, the server will take care.
But anyway we can keep this implementation as another fail-safe. Up to you

Reply:
Lets leave it as it is just to be on the safe side